codebuild_specs/aggregate_e2e_reports.yml

```diff
@@ -10,9 +10,13 @@ phases:
       - ts-node ./wait-for-all-codebuild.ts $CODEBUILD_RESOLVED_SOURCE_VERSION ../$WAIT_FOR_IDS_FILE_PATH $PROJECT_NAME
       - cd ..
       - source ./shared-scripts.sh && _downloadReportsFromS3 $CODEBUILD_SOURCE_VERSION
+      - mkdir artifacts && echo "empty artifact" > artifacts/empty-artifact.txt
```

Comment:
Is this needed for cleanup script or just an unrelated change?

Reply:
just needed any artifact, as the cleanup job depends on the aggregate job to finish; Codebuild requires 'some' file to be produced in order to depend on a job